### ISSUE TITLE
Why the image_set in the config file is train for val?

### ISSUE THREAD

**Issue by johnnylu305** [ziqi-jin/finetune-anything#48]
Hi, I have a few questions about the config file. 

1. Why the image_set in the [config](https://github.com/ziqi-jin/finetune-anything/blob/main/config/semantic_seg.yaml#L75) file is train for val? Will this cause the validation benchmark to run on the training set instead of the validation set?

2. Did the [mIoU 78](https://github.com/ziqi-jin/finetune-anything/issues/22) you got based on the current version of FA? If not, can you update the code and provide some insights about the new version of FA?

Thanks.
